packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1367,27 +1368,30 @@ export class Combobox
     chip?.setFocus();
   }
 
-  private scrollToActiveItem(): void {
-    const activeItem = this.filteredItems[this.activeItemIndex];
+  private scrollToActiveOrSelectedItem(selected = false): void {
+    const item =
+      selected && this.selectedItems && this.selectedItems.length
+        ? this.selectedItems[0]
+        : this.filteredItems[this.activeItemIndex];
 
-    if (!activeItem) {
+    if (!item) {
       return;
     }
 
-    const height = this.calculateScrollerHeight(activeItem);
+    const height = this.calculateScrollerHeight(item);
```

Comment:
@josercarcamo instead of calculating the height, can you try the following:

```tsx
item.scrollIntoView({ block: "nearest" });
```

I'm not sure calculating the offset is necessary.